src/SignalR/common/Http.Connections/test/HttpConnectionDispatcherTests.cs

```diff
@@ -1347,358 +1347,6 @@ public async Task TransferModeSet(HttpTransportType transportType, TransferForma
             }
         }
 
-        [Fact]
-        public async Task UnauthorizedConnectionFailsToStartEndPoint()
-        {
-            using (StartVerifiableLog())
-            {
-                var manager = CreateConnectionManager(LoggerFactory);
-                var connection = manager.CreateConnection();
-                connection.TransportType = HttpTransportType.LongPolling;
-                var dispatcher = new HttpConnectionDispatcher(manager, LoggerFactory);
-                var context = new DefaultHttpContext();
-                var services = new ServiceCollection();
-                services.AddOptions();
-                services.AddSingleton<TestConnectionHandler>();
-                services.AddAuthorization(o =>
-                {
-                    o.AddPolicy("test", policy => policy.RequireClaim(ClaimTypes.NameIdentifier));
-                });
-                services.AddAuthenticationCore(o =>
-                {
-                    o.DefaultScheme = "Default";
-                    o.AddScheme("Default", a => a.HandlerType = typeof(TestAuthenticationHandler));
-                });
-                services.AddLogging();
-                var sp = services.BuildServiceProvider();
-                context.Request.Path = "/foo";
-                context.Request.Method = "GET";
-                context.RequestServices = sp;
-                var values = new Dictionary<string, StringValues>();
-                values["id"] = connection.ConnectionId;
-                var qs = new QueryCollection(values);
-                context.Request.Query = qs;
-
-                var builder = new ConnectionBuilder(sp);
-                builder.UseConnectionHandler<TestConnectionHandler>();
-                var app = builder.Build();
-                var options = new HttpConnectionDispatcherOptions();
-                options.AuthorizationData.Add(new AuthorizeAttribute("test"));
-
-                // would get stuck if EndPoint was running
-                await dispatcher.ExecuteAsync(context, options, app).OrTimeout();
-
-                Assert.Equal(StatusCodes.Status401Unauthorized, context.Response.StatusCode);
-            }
-        }
-
-        [Fact]
-        public async Task AuthenticatedUserWithoutPermissionCausesForbidden()
```

Comment:
Do we still have end-to-end coverage for cases like this?

Reply:
We do have some end-to-end tests for auth, for example: https://github.com/aspnet/AspNetCore/pull/10471/files#diff-c65c848cc599ae1724a72656ff60bedaR453

But we don't need to verify it as much because we now rely on the underlying auth system instead of implementing our own version.

Reply:
Do we still manually verify authorization for hub method invocations to account for method-level auth attributes over a WebSocket for example? Where is that?

Reply:
Yes we still do. This PR doesn't touch any of that.

Test located [here](https://github.com/aspnet/AspNetCore/blob/a26c4e936b1a775b890a8fb2d33fdefcece60518/src/SignalR/server/SignalR/test/HubConnectionHandlerTests.cs#L2156)

Reply:
Yeah, I can't speak to the specific tests that are being removed, but I'd strongly encourage testing a variety of auth scenarios and interactions as a user would write them.